core/src/main/java/org/commonjava/maven/ext/manip/state/DependencyState.java

```diff
@@ -41,6 +41,11 @@
     private static final String DEPENDENCY_EXCLUSION_PREFIX = "dependencyExclusion.";
 
     /**
+     * Merely an alias for {@link DependencyState#DEPENDENCY_EXCLUSION_PREFIX}
+     */
+    private static final String DEPENDENCY_OVERRIDE_PREFIX = "dependencyOverride.";
+
+    /**
```

Comment:
I'd kind of like to have it the other way around, so "dependencyExclusion" is an alias for "dependencyOverride" even though it doesn't affect the functionality.  Based on the fact that the docs repeatedly use the term "override" in the feature descriptions, it seems like "dependencyOverride" should be the recommended parameter.